scripts/quests/hiddenQuests/New_Character_Cutscenes.lua

```diff
@@ -0,0 +1,341 @@
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_2)
+
+                    return { 500, -1, cutsceneFlags }
+                end
+            },
+
+            onEventFinish =
+            {
+                [500] = function(player, csid, option, npc)
+
+                    local ID = zones[player:getZoneID()]
+                    -- If you don't get the coupon, tough luck. Retail doesn't give you a chance to get it again.
+                    npcUtil.giveItem(player, xi.items.ADVENTURERS_COUPON)
+                    player:messageText(player, ID.text.MAP_MARKER_TUTORIAL)
+
+                    player:setPos(-100, -8, -125, 224)
+                    player:setHomePoint()
+
+                    quest:setVar(player, 'notSeen', 0)
+                end,
+            },
+        },
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    local cutsceneFlags = 0
+
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_1)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.NO_PCS)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.NO_NPCS)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_2)
+
+                    return { 531, -1, cutsceneFlags }
+                end
+            },
+
+            onEventUpdate =
+            {
+                [531] = function(player, csid, option, npc)
+
+                    -- param 7 is checked in the event according to decomp, but it was never set on Hume F starting a new character.
+                    -- param 7 was 0x20012 on a galka changing nations.
+                    player:updateEvent({ [7] = 0 })
+                end,
+            },
+
+            onEventFinish =
+            {
+                [531] = function(player, csid, option, npc)
+                    local ID = zones[player:getZoneID()]
+                    player:messageText(player, ID.text.MAP_MARKER_TUTORIAL)
+                    -- If you don't get the coupon, tough luck. Retail doesn't give you a chance to get it again.
+                    npcUtil.giveItem(player, xi.items.ADVENTURERS_COUPON)
+
+                    -- The HP position is slightly different than the CS exit.
+                    player:setPos(-40.0, -5, 100, 64)
+                    player:setHomePoint()
+                    player:setPos(-40.611, -5, 102.5, 57) -- Move back to CS exit position
+
+                    quest:setVar(player, 'notSeen', 0)
+                end,
+            },
+        },
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    local cutsceneFlags = 0
+
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_1)
```

Comment:
An idea, not a request:

I don't know if it would look/read better, but I _think_ `bit.bor` is variadic:

```
local cutsceneFlags = bit.bor(
    xi.cutscenes.params.UNKNOWN_2,
    xi.cutscenes.params.NO_PCS,
    xi.cutscenes.params.NO_NPCS,
    xi.cutscenes.params.UNKNOWN_2,
)
```

Reply:
It is variadic, had stared at the formatting for it but didn't know the best way to do it. thanks